packages/vats/src/core/chain-behaviors.js

```diff
@@ -32,6 +33,58 @@ const { keys } = Object;
 
 const NUM_IBC_PORTS = 3;
 
+/**
+ * This is the code triggered by `agd tx gov submit-proposal swingset-core-eval
+ * permit.json code.js`.  It is the "big hammer" governance that allows code.js
+ * access to all powers permitted by permit.json.
+ *
+ * @param {BootstrapPowers} allPowers
+ */
+export const makeCoreEval = async allPowers => {
+  // We need all of the powers to be available to the evaluator, but we only
+  // need the bridgeManager to install our handler.
+  const {
+    consume: { bridgeManager: bridgeManagerP },
+  } = allPowers;
+  const bridgeManager = await bridgeManagerP;
+  if (!bridgeManager) {
+    // Not running with a bridge.
+    return;
+  }
+
+  // Register a coreEval handler over the bridge.
+  const handler = Far('coreHandler', {
+    async fromBridge(_srcID, obj) {
+      switch (obj.type) {
+        case 'CORE_EVAL': {
+          const { evals } = obj;
+          return Promise.all(
+            evals.map(({ json_permits: permit, js_code: code }) =>
+              // Run in a new turn to avoid crosstalk of the evaluations.
+              Promise.resolve().then(() => {
+                const powers = extractPowers(permit, allPowers);
+
+                // Inspired by ../repl.js:
+                const globals = harden({
+                  ...farExports,
+                  console,
```

Comment:
`console`... hm... I guess that's reasonable.

How about `assert` while we're at it?

Reply:
> How about `assert` while we're at it?

Added here and in `repl.js` for the pleasure of the ag-solo REPL.